### ISSUE TITLE
Why the loss of D quickly go down to zero?

### ISSUE THREAD

**Issue by Lycheezx** [znxlwm/pytorch-MNIST-CelebA-GAN-DCGAN#12]
Hello
I ran your code of DCGAN implementation on dataset of MNIST but the quality of the generated images were poor. I have already tried to reduce the learning rate but it didn't work and the result was a far cry from yours.  I am new to GAN and feel really confused. Did you change your parameter settings or do some other adjustments? Hope to get some advice from you, thank you very much!

**Comment by yuchenlichuck** [comment]
Same to you~

**Comment by Newbeeer** [comment]
Me too.  I guess another repo https://github.com/eriklindernoren/PyTorch-GAN makes life much easier :)

**Comment by omid-ghozatlou** [comment]
The results are really poor. I recommend this repo too.

> Me too. I guess another repo https://github.com/eriklindernoren/PyTorch-GAN makes life much easier :)

**Comment by Tang-08080103** [comment]
The MNIST result of DCGAN is very poor, I thought it was a problem with my environment. This is too bad.

**Comment by lllyhgit** [comment]
me too!
